jsonnet/telemeter/rules.libsonnet

```diff
@@ -35,6 +35,12 @@
                 topk by (_id) (1, max by (_id, managed, ebs_account, internal) (label_replace(label_replace((subscription_labels{support=~"Standard|Premium|Layered"} * 0 + 1) or subscription_labels * 0, "internal", "true", "email_domain", "redhat.com|(.*\\.|^)ibm.com"), "managed", "", "managed", "false")) + on(_id) group_left(version) (topk by (_id) (1, 0*cluster_version{type="current"})))
               |||,
             },
+            {
+              record: 'subscription_labels:not_redhat_not_ibm',
+              expr: |||
+                topk by (_id) (1, 0 *subscription_labels{email_domain!~"redhat.com|(^|.*\\.)ibm.com"})
```

Comment:
This is what I stole from Clayton IIRC, and maybe we could add another recording rule for the first part as that is something they use from what I could tell:
```
(
  ((time() - topk by (_id) (1,
    label_replace(
      label_replace(cluster_version{type="current"}, "version", "4.$1-0.$2", "version", "4\\.(\\d+\\.\\d+)-0.(ci|nightly|okd).*"),
      "version",
      "ci-pull-requests",
      "version",
      "0.0.1-.*"
    )
  )))
  + on (_id) group_left() (0 * topk by (_id) (1, subscription_labels{email_domain!~"redhat.com|(^|.*\\.)ibm.com"})) + 0
))
```

Reply:
that's also a good one, let me add that now already